### PR TITLE
JBPM-5233: Data source management

### PR DESCRIPTION
**Pull request by wmedvede**
- Adds the data source managment modules to the kie-bom
    - Adds some dependencies required by the data source management modules to the kie-parent

### CODE REVIEW

**Comment by wmedvede** [comment]
@manstis @csadilek 

Hello Mike, Christian,
This is the data sources management PR related to droolsjbpm-build-boostrap. Would you mind take a look.

Thanks!!
Walter.

File: pom.xml [hunk 1]
```diff
@@ -1688,7 +1688,37 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- Data source management libraries -->
+      <dependency>
+        <groupId>org.jboss</groupId>
+        <artifactId>jboss-dmr</artifactId>
+        <!-- corresponds to WF 10 distribution -->
+        <version>1.3.0.Final</version>
+      </dependency>
```
Comment (csadilek) [inline pom.xml]: We should define version properties for all these.

**Comment by wmedvede** [comment]
@csadilek properties added.

**Comment by psiroky** [comment]
@wmedvede could you please also create PRs for IP BOM? It's fine to have them temporarily here, but longer we need to make sure they end-up in IP BOM and then we could remove the from here.

**Comment by csadilek** [comment]
+1

**Pull request merged**